### ISSUE TITLE
Docs: Add separate sections for scripts

### ISSUE THREAD

**Issue by Jay-Karia** [eslint/eslint#18768]
### Docs page(s)

This is one of the page that needs to be changed:
https://eslint.org/docs/latest/use/getting-started#quick-start

### What documentation issue do you want to solve?

In this page the script to init eslint includes pnpm, yarn and npm in one section.
It is difficult to copy the desired command.

### What do you think is the correct solution?

We can divide them into their respective sections and add a copy button.

Here's an example:

```shell copy
npm script
```
or
```shell copy
pnpm script
```
or
```shell copy
ynpm script
```

This could improve DX

### Participation

- [X] I am willing to submit a pull request for this change.

**Comment by Tanujkanti4441** [comment]
Let's wait for issue #18769 to be accepted for this.

**Comment by Tanujkanti4441** [comment]
Accepted! a PR is welcome.

**Comment by nzakas** [comment]
:-1: I'm not in favor of this change. It makes the installation section way too long. I think the text is fine the way it is.

**Comment by harish-sethuraman** [comment]
I'm also not in favor of this change. As discussed earlier adding tab would be better than this or lets leave it as such.

**Comment by Jay-Karia** [comment]
So I am closing this.
Should I open a new issue for tabs ?

**Comment by amareshsm** [comment]
> So I am closing this. Should I open a new issue for tabs?

Yes, it's good to open a new issue for tabs.
